### PR TITLE
feat(pkger): add service logging

### PR DESCRIPTION
**Pull request by jsteenb2**
was helping @russorat debugging issue he is seeing and wanted to enhance the data the pkger service provides. 

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: pkger/service_logging.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		fields := s.summaryLogFields(pkg.Summary())
+		s.logger.Info("pkg create", append(fields, dur)...)
+	}(time.Now())
+	return s.next.CreatePkg(ctx, setters...)
+}
+
+func (s *loggingMW) DryRun(ctx context.Context, orgID, userID influxdb.ID, pkg *Pkg) (sum Summary, diff Diff, err error) {
+	defer func(start time.Time) {
+		dur := zap.Duration("took", time.Since(start))
+		if err != nil {
```
Comment (kelwang) [inline pkger/service_logging.go]: not sure this will work, have you test it out? Usually, i would pass an *error pointer as a parameter of your defer func

Comment (jsteenb2) [reply pkger/service_logging.go]: this works b/c the enclosure isn't forcing the assignment. When you pass an error as a func arg, you'll need to that as `*error`

Comment (kelwang) [reply pkger/service_logging.go]: nice

**Comment by jsteenb2** [comment]
@bthesorceror / @kelwang  I went ahead and added another middlware for tracing functionality

**Pull request merged**